src/pymor/bindings/dunegdt.py

```diff
@@ -0,0 +1,418 @@
+        grid
+            The dune grid associated with space (assumed to be equidistant!).
+        """
+        def __init__(self, space, grid):
+            assert grid.dimension == 1
+            assert space.dimDomain == 1
+            # build pyMOR grid
+            centers = np.array(grid.centers(1), copy=False).ravel()
+            domain = (centers[0], centers[-1])
+            num_intervals = grid.size(0)
+            pymor_grid = OnedGrid(domain, num_intervals)
+            if space.max_polorder == 0:
+                assert space.num_DoFs == grid.size(0)
+                self.visualizer = OnedVisualizer(pymor_grid, codim=0)
+            elif space.min_polorder == space.max_polorder == 1:
+                assert space.num_DoFs == grid.size(1)
+                self.visualizer = OnedVisualizer(pymor_grid, codim=1)
+            else:
+                # TODO: add P1 interpolation?
+                raise NotImplementedError('Not available for higher polynomial orders!')
+            self.__auto_init(locals())
+
+        def visualize(self, U, m, **kwargs):
+            # convert to NumpyVectorArray
+            U_np = NumpyVectorSpace(U.dim).zeros(len(U))
+            for ii, u_dune in enumerate(U._list):
+                u_np = np.array(u_dune.impl, copy=False)
+                U_np._array[ii, :] = u_np[:]
+            return self.visualizer.visualize(U_np, None, **kwargs)
+
+
+    class DuneGDT1dMatplotlibVisualizer(ImmutableObject):
+        """Visualize a dune-gdt discrete function using paraview.
+
+        Parameters
+        ----------
+        space
+            The dune-gdt space for which we want to visualize DOF vectors.
+        """
+        def __init__(self, space):
+            assert space.dimDomain == 1
+            assert space.max_polorder == 1
+            self.interpolation_points = space.interpolation_points()
+            self.__auto_init(locals())
+
+        def visualize(self, U, m, title=None, legend=None, separate_colorbars=False,
+                      rescale_colorbars=False, block=None, filename=None, columns=2):
+            assert isinstance(U, ListVectorArray)
+            assert len(U) == 1
+            U = U._list[0]
+            assert isinstance(U, DuneXTVector)
+            X = np.array(self.interpolation_points, copy=False)
+            Y = np.array(U.impl, copy=False)
+            name = legend if legend else 'STATE'
+            plt.plot(X, Y, label=name)
+            if title:
+                plt.title(title)
+            plt.legend()
+            plt.plot()
+
+
+    class DuneGDTParaviewVisualizer(ImmutableObject):
+        """Visualize a dune-gdt discrete function using paraview.
+
+        Parameters
+        ----------
+        space
+            The dune-gdt space for which we want to visualize DOF vectors.
+        """
+
+        def __init__(self, space):
+            self.__auto_init(locals())
+
+        def visualize(self, U, m, title=None, legend=None, separate_colorbars=False,
```

Comment:
This would need a general overhaul to match the current meshio usage like in the fenics visualizer.

Reply:
I don't understand yet what your comment applies to

Reply:
Visualizers use mesio internally to represent data, this allows fenics, ngsolve, etc as input.  See here: https://github.com/pymor/pymor/blob/pyvista_trial/src/pymor/bindings/fenics.py#L475
I forgot however that this is not in main yet. So disregard my comment, if this visualizers works as atm.

Reply:
They do, but I'll gladly use/share other code once something is in main...